bro_connector/frd/models.py

```diff
@@ -290,4 +290,31 @@ class FormationresistanceRecord(FRDRecord):
     class Meta:
         managed = True
         db_table = 'frd"."formationresistance_record'
-        verbose_name_plural = "Formationresistance Records"
\ No newline at end of file
+        verbose_name_plural = "Formationresistance Records"
+
+class FrdSyncLog(models.Model):
+    synced = models.BooleanField(default=False)
+    date_modified = models.CharField(max_length=254, null=True, blank=True)
+    event_type = models.CharField(
+        choices=EVENT_TYPE_CHOICES,
+        blank=False,
+        null=True,
+        max_length = 25,
+    )
+    frd = models.ForeignKey(FormationResistanceDossier, on_delete = models.CASCADE, null = True, blank = True)
+    frd_bro_id = models.CharField(max_length=254, null=True, blank=True)
+    process_status = models.CharField(max_length=254, null=True, blank=True)
+    comment = models.CharField(max_length=10000, null=True, blank=True)
+    xml_filepath = models.CharField(max_length=254, null=True, blank=True)
+    delivery_status = models.IntegerField(choices=LEVERINGSTATUS_CHOICES, null=True, blank=True, default = 0)
+    delivery_status_info = models.CharField(max_length=254, null=True, blank=True)
+    delivery_id = models.CharField(max_length=254, null=True, blank=True)
+    
+        
+    def __str__(self):
+        return f"{self.event_type}_{self.frd.object_id_accountable_party}_log"
```

Comment:
Oei, dit klinkt misschien als de meest onschuldige regel van het hele pull request, maar: *gevaar*.

- `self.frd` mag leeg zijn => `AttributeError` zodra je dit object ergens print (of ergens in de admin in een lijstje toont).
- Je refereert naar een ander object. Dus als je een lijstje logs ergens opbouwt in een query (zeg 400 stuks) en je laat het zien (bijvoorbeeld in zo'n `list_filter`), dan krijg je 400 losse sql queries extra om de `.frd` op te halen. Tenzij je overal `.select_related` goed configureert.